src/content/changelogs/fundamentals.yaml

```
@@ -5,6 +5,12 @@ productLink: "/fundamentals/"
 productArea: Core platform
 productAreaLink: /fundamentals/reference/changelog/platform/
 entries:
+  - publish_date: "2024-12-16"
+    title: Cloudflare API docs are now automatically generated
+    description: | - 
+      Cloudflare's API documentation is now being automatically generated based on OpenAPI Schemas, and we have retired our old documentation. The move to OpenAPI Schemas allows us to ensure greater consistency and quality across our API documentation. The documentation now also includes examples of how to call the API using curl, Typescript, Python, and Go. 
```

non-blocking: perhaps a more generic term for our SDKs to make this more applicable for longer?

```suggestion
      Cloudflare's API documentation is now being automatically generated based on OpenAPI Schemas, and we have retired our old documentation. The move to OpenAPI Schemas allows us to ensure greater consistency and quality across our API documentation. The documentation now also includes examples of how to call the API using curl and our SDKs. 
```

---

Sure, but I'm going to Oxford that comma back in 😉